Client/Application/AppDelegate.swift

```diff
@@ -455,6 +460,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         // is that this method is only invoked whenever the application is entering the foreground where as
         // `applicationDidBecomeActive` will get called whenever the Touch ID authentication overlay disappears.
         self.updateAuthenticationInfo()
+        /*
+         Ecosia
```

Comment:
just an aesthetical side note - Is it intended that there is an extra space before Ecosia and the closing comment (*/)?  I also wonder if we could put it in one line instead `*/ Ecosia */` or just `// Ecosia`for single line comments? for multiline I agree to use block comments but would prefer single line comments for single lines. what do you think?

Reply:
I have no strong opinion about it either way, though I still fail to see the real value of adding comments where we make changes, not sure yet if it is going to help us, I just think that if we are going to do it we should try to be consistent, either all comments are // or all comments are /**/, otherwise we might be defeating the purpose of being easily spotted and grouped, or is there any other purpose for it @ecotopian ?

Reply:
I also don't have a strong opinion, so let's try to find a way that fits us best in this proj :)  

the purpose originates from our experience on Chromium where it helps us understanding and solving merge conflicts on upgrades. So I would suggest we add/use comments when
- we block comment bigger chunks (to have less conflict then by deleting) 
- when the comment helps to understand why we modified the code

I'd say for the Analytics it is obvious and we don't need comments there at all, agree?

And for consistency let's add comments as block like starting in a single line like `/* Ecosia ... `so we can consistently find them again. Does that make sense?

Reply:
Still not sure of the value of comments as a tool to track changes, but we can try.

Reply:
it's more about understanding rather than tracking.. so if you feel like a comment is needed to help understanding why a modification is done, feel free to add it.. if not feel free to leave it :)

Reply:
I meant more the other scenario, when removing chunks of code. Deleting vs Commenting, I wish there was even a third option but I can't think of anything at the moment.